clang/lib/DPCT/Error.cpp

```diff
@@ -38,6 +38,12 @@ void ShowStatus(int Status, std::string Message) {
     StatusString = "Error: Path for CUDA header files specified by "
                    "--cuda-include-path is invalid.";
     break;
+  case MigrationErrorInvalidChangeFilenameExtension:
+    StatusString =
+        "Error: Extension specified by --change-filename-extension is "
+        "invalid.\n"
+        "Only a '.' followed by alphabetical characters and digits is allowed.";
```

Comment:
a little update:  digits is not allowed according to DPCT.cpp line L672 code logic.

Reply:
fixed